recipes/umi_tools/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
 test:
   commands:
     - umi_tools --help
+    - test -f $SP_DIR/umi_tools/_dedup_umi.c
```

Comment:
True, but we are also testing in a Docker container without conda installed, so this magic variable does not exists there. But your tests are failing earlier.
Can you try with these versions: https://github.com/bioconda/bioconda-utils/blob/master/bioconda_utils/bioconda_utils-requirements.txt#L5